tests/tokio.rs

```diff
@@ -0,0 +1,93 @@
+#[cfg(feature = "tokio-traits")]
+mod tokiotest {
+    macro_rules! rtt {
```

Comment:
I wonder if there is a good way for us to share these macros between the mods. I _think_ can do it by adding the trait to use as an argument to the macro... And maybe also the macro annotation to use for the test.